src/simulation/environment/eclipse/eclipse.h

```diff
@@ -51,6 +51,7 @@ class Eclipse: public SysModel {
     std::vector<ReadFunctor<SCStatesMsgPayload>> positionInMsgs;  //!< vector of msgs for each spacecraft position state for which to evaluate eclipse conditions.
     std::vector<Message<EclipseMsgPayload>*> eclipseOutMsgs;//!< vector of eclispe output msg names
     BSKLogger bskLogger;                        //!< BSK Logging
+    double REQ_CUSTOM;  //!< Custom radius, km
```

Comment:
Please make this meters, not km if the user is setting up a custom radius.  It is up to the user to ensure this has the proper units.